docs/pages/faq.mdx

```diff
@@ -120,17 +120,20 @@ Teleport provides three offerings:
 
 Teleport provides security-critical support for the current and two previous releases. With our typical release cadence, this means a release is usually supported for 9 months.
 
-| Release | Long Term Support | Release Date | Min tsh version |
-| - | - | - | - |
-| 6.2 | No | May 21th, 2021 | 3.0.0 |
-| 6.1 | No | April 9th, 2021 | 3.0.0|
-| 6 | Yes | March 4th, 2021 | 3.0.0 |
-| 5.0 | Yes | November 24th, 2020 | 3.0.0 |
-| 4.4 | Yes | October 20th, 2020 | 3.0.0 |
-| 4.3 (EOL) | Yes | July 8th, 2020 | 3.0.0 |
-| 4.2 (EOL) | Yes | December 19th, 2019 | 3.0.0 |
-| 4.1 (EOL) | Yes | October 1st, 2019 | 3.0.0 |
-| 4.0 (EOL) | Yes | June 18th, 2019 | 3.0.0 |
+| Release | Supported | Release Date        |
+| --------|-----------| --------------------|
+| 9.x.x   | Yes       | March 11th, 2022    |
```

Comment:
Why does this say `9.x.x`? We're not releasing `9.x.x` on March 11, we're releasing 9.0.0.

Reply:
I think this refers to the 9.x.x series. I just made it consistent with how we did it in the past.